projects/ngx-monaco-tree/src/lib/monaco-tree-file/monaco-tree-file.component.ts

```diff
@@ -49,6 +50,12 @@ export class MonacoTreeFileComponent implements OnChanges {
 	position: [number, number]|undefined = undefined;
 
 
+  // TODO use computed once we use input-signal instead of input-decorator
+  get sortedTree() {
+    return this.content?.slice().sort(sortTreeElement)
```

Comment:
the out value will be different on each call of this function which will probably mess with the change detection in some ways
I understand this will be fix with the `//TODO` above but as a temporary solution this seems not ideal

Reply:
Indeed, I will wait to move to input-signal before introducing that feature ! :)